src/cljs/rems/fields.cljs

```diff
@@ -73,7 +88,9 @@
         optional (:field/optional opts)
         value (:field/value opts)
         previous-value (:field/previous-value opts)
-        max-length (:field/max-length opts)]
+        max-length (:field/max-length opts)
+        info-text (localized (:field/info-text opts))
+        collapse-aria-label (str (text :t.create-form/collapse-aria-label) (linkify raw-title))]
```

Comment:
looks like you could just use `title` here instead of recomputing `(linkify raw-title)`